configuration/jms-setup-wildfly.cli

```diff
@@ -0,0 +1,9 @@
+connect
+batch
+
+/subsystem=messaging/hornetq-server=default/address-setting=jms.queue.TriggerMetricCollectionQueue:add(redelivery-delay=1000, max-delivery-attempts=300)
```

Comment:
@lfryc the file needs to be included in the DIST bundle (via our maven assembly)

Something else, is that in database we already some .cli scripts - is this something we can combine?
and perhaps just rename them?)